scripts/bug_classifier.py

```diff
@@ -46,8 +47,10 @@ def classify_bugs(model_name, classifier, bug_id):
     model_class = get_model_class(model_name)
     model = model_class.load(model_file_name)
 
+    bug_id = int(bug_id)
```

Comment:
Add `type=int` to the `--bug-id` argument instead.

Reply:
Fixed